src/components/StatusBadge/StatusBadge.tsx

```diff
@@ -0,0 +1,52 @@
+import * as React from "react";
+import bem from "../../utils/bem";
+
+export interface StatusBadgeProps {
+    /** BlockName for use with BEM. See how to work with blockNames and BEM here: http://getbem.com/introduction/ */
+    blockName?: string;
+    /** ClassName that appears in addition to "status-badge" */
+    className?: string;
+    /** Inner text of the element */
+    text?: string;
+    /** Type of badge **/
+    type?: string;
```

Comment:
I'd use a string literal here to expose the [types available](https://www.figma.com/file/qShodlfNCJHb8n03IFyApM/Main?node-id=11256%3A0) to developers more clearly. See more info [here](https://github.com/typescript-cheatsheets/react/blob/cc2ec67e0b36b7e838df8f2a54b4cd84df6c78cd/README.md#basic-prop-types-examples).

`type?: ""waiting" | "success";`